docs/usage/sat_and_iuf.md

```diff
@@ -0,0 +1,259 @@
+# SAT and IUF
+
+The Install and Upgrade Framework (IUF) provides commands which install,
+upgrade, and deploy products on systems managed by CSM with the help of `sat
+bootprep`. For more information on IUF, see the [IUF
+section](https://cray-hpe.github.io/docs-csm/en-14/operations/iuf/iuf/) of the
+[Cray System Management Documentation](https://cray-hpe.github.io/docs-csm/).
+
+## Edit Default Bootprep Input Files
+
+You might need to customize the default bootprep input files delivered by
+the HPC CSM Software Recipe for your system. Here are some examples of how to
+edit the files when using them with either `sat bootprep` or IUF.
+
+### Edit Default Branch Names
+
+Before running `sat bootprep`, HPE recommends reading the bootprep input files
+and paying specific attention to the `branch` parameters. Some HPE Cray EX
+products require system-specific changes on a working branch of VCS. For these
+products, the default bootprep input files assume certain naming conventions for
+the VCS branches. The files refer to a particular branch of a product's
+configuration management repository.
+
+Thus, it is important to confirm that the bootprep input files delivered by the
+HPC CSM Software Recipe match the actual system branch names. For example, the
+COS product's CFS configuration layer is defined as follows in the default
+`management-bootprep.yaml` bootprep input file.
+
+```yaml
+- name: cos-ncn-{{cos.working_branch}}
+  playbook: ncn.yml
+  product:
+    name: cos
+    version: "{{cos.version}}"
+    branch: "{{cos.working_branch}}"
+```
+
+The default file is assuming that system-specific Ansible configuration changes
+for the COS product in VCS are stored in a branch named
+`"{{cos.working_branch}}"`. SAT bootprep looks for a branch name (with all
+variable values resolved) from which to create CFS configuration layers. For
+example, `integration-2.5.99` for COS 2.5.99.
+
+You can create VCS working branches that are not the default bootprep input file
+branch names. For example, to use the branch `integration` for COS instead of
+`"{{cos.working_branch}}"`, edit the bootprep input file so it reads as
+follows.
+
+```yaml
+- name: cos-ncn-{{cos.working_branch}}
+  playbook: ncn.yml
+  product:
+    name: cos
+    version: "{{cos.version}}"
+    branch: integration
+```
+
+### Edit Default Management CFS Configuration Names
+
+The default bootprep input file for management NCNs (`management-bootprep.yaml`)
+uses the value `management-{{recipe.version}}` as the name of the CFS
+configuration. This value uses a Jinja2 template to include the HPC CSM Software
+Recipe version in the name of the CFS configuration. For example, when `sat
+bootprep` is run against the default bootprep input file in HPC CSM Software
+Recipe version `23.05`, a configuration named `management-23.05` is created.
```

Comment:
There have been some changes since I wrote this line. Because we need to conform to [SemVer](https://semver.org/) formatting, we do not zero-pad the second component of the version string, and we always include the third component, set to zero for the first release of a recipe. As far as I know, we still do use the two-component version with a zero-padded month in documentation and release communications. Last I heard, we are also planning to release Alice in April, so it will be versioned as 23.04 in release communications, and this version will be 23.4.0. So you could change this line to:

```suggestion
Recipe version 23.04, a configuration named `management-23.4.0` is created.
```

Reply:
Or, if we want to only use the SemVer formatted version, we could say:

```suggestion
Recipe version `23.4.0`, a configuration named `management-23.4.0` is created.
```

I can't decide which is better.

Reply:
@haasken-hpe, I believe Sarah's new _HPE Cray EX System Software Stack Installation and Upgrade Guide for CSM
(S-8052)_ is still using the "23.04" format. For that reason, I'm going to go with your first suggestion and update throughout.

To focus this PR more on the SAT and IUF relationship, I'm going to add this change to PR #88 instead. PR #88 will be merged shortly, and I'll rebase our other PRs to include it.

Reply:
Note that the team ultimately decided to remove this section and place it in the CSM docs instead.